testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/KeycloakRealmImportOperatorProvisionerTest.java

```diff
@@ -0,0 +1,373 @@
+			return POSTGRESQL_USER;
+		}
+
+		@Override
+		public String getPassword() {
+			return POSTGRESQL_PASSWORD;
+		}
+
+		@Override
+		public String getDbName() {
+			return POSTGRESQL_DATABASE;
+		}
+	};
+	private static final PostgreSQLImageOpenShiftProvisioner POSTGRESQL_IMAGE_PROVISIONER = new PostgreSQLImageOpenShiftProvisioner(
+			pgSQLApplication);
+
+	private static KeycloakRealmImportOperatorProvisioner initializeOperatorProvisioner(final Keycloak keycloak,
+			final String appName) {
+		KeycloakRealmImportOperatorProvisioner operatorProvisioner = new KeycloakRealmImportOperatorProvisioner(
+				new KeycloakRealmImportOperatorApplication() {
+
+					@Override
+					public Keycloak getKeycloak() {
+						return keycloak;
+					}
+
+					@Override
+					public String getName() {
+						return appName;
+					}
+				});
+		return operatorProvisioner;
+	}
+
+	private String name;
+	private String realmName;
+
+	private static final Map<String, String> matchLabels = new HashMap<>();
+
+	@BeforeAll
+	public static void createOperatorGroup() throws IOException {
+		matchLabels.put("app", "sso");
+		IntersmashExtension.operatorCleanup();
+		// create operator group - this should be done by InteropExtension
+		OpenShifts.adminBinary().execute("apply", "-f", OperatorGroup.SINGLE_NAMESPACE.save().getAbsolutePath());
+	}
+
+	@BeforeEach
+	public void cleanup() throws IOException {
+		if (!Objects.isNull(KEYCLOAK_OPERATOR_PROVISIONER))
+			KEYCLOAK_OPERATOR_PROVISIONER.unsubscribe();
+	}
+
+	@AfterAll
+	public static void removeOperatorGroup() {
+		OpenShifts.adminBinary().execute("delete", "operatorgroup", "--all");
+		if (!Objects.isNull(KEYCLOAK_OPERATOR_PROVISIONER))
+			KEYCLOAK_OPERATOR_PROVISIONER.dismiss();
+		POSTGRESQL_IMAGE_PROVISIONER.undeploy();
+		POSTGRESQL_IMAGE_PROVISIONER.postUndeploy();
+	}
+
+	@AfterEach
+	public void customResourcesCleanup() {
+		if (!Objects.isNull(KEYCLOAK_OPERATOR_PROVISIONER)) {
+			// delete keycloaks
+			KEYCLOAK_OPERATOR_PROVISIONER.keycloakClient().list().getItems().stream()
+					.map(resource -> resource.getMetadata().getName()).forEach(name -> KEYCLOAK_OPERATOR_PROVISIONER
+							.keycloakClient().withName(name).withPropagationPolicy(DeletionPropagation.FOREGROUND).delete());
+			// delete realms
+			KEYCLOAK_OPERATOR_PROVISIONER.keycloakRealmImportClient().list().getItems()
+					.stream()
+					.map(resource -> resource.getMetadata().getName()).forEach(name -> KEYCLOAK_OPERATOR_PROVISIONER
+							.keycloakRealmImportClient().withName(name).withPropagationPolicy(DeletionPropagation.FOREGROUND)
+							.delete());
+		}
+	}
+
+	/**
+	 * This test case creates and validates a basic {@link Keycloak} CR
+	 *
+	 * This is not an integration test, the goal here is to assess that the created CRs are configured as per the
+	 * model specification.
+	 *
+	 * See
+	 * <br> - https://github.com/keycloak/keycloak-operator/tree/master/deploy/examples/keycloak
+	 */
+	@Test
+	public void exampleSso() {
+		try {
```

Comment:
Here and in the other test method we should use this pattern: https://github.com/Intersmash/intersmash/blob/main/testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/InfinispanOperatorProvisionerTest.java#L130

i.e.:
```
subscribe
try {
  // create CR
  // verify CR
} finally {
  unsubscribe
}
```

Reply:
Well, thare are a couple of issues:
1. `configure` shouldn't be here as said above
2. the `subscribe` method should be called before the `try` since it would make sense for the `unsubscribe` in the `finally` block to be executed.

Reply:
done, hope it's done the way what we discussed!